### ISSUE TITLE
Panoptes backend for sports domain

### ISSUE THREAD

**Issue by pasky** [AugurProject/argus#5]
This has a lot of open questions, e.g. how to precisely specify the match / race.

Sources:
  * [x] Investigate Yahoo Fantasy - bad, see wiki
  * [X] Investigate scraping NBCSports - against ToS
  * [ ] Investigate Wolfram Alpha
  * [ ] Investigate http://www.cricapi.com/ for Cricket, specifically

Domains:
  * [ ] soccer
  * [ ] tennis
  * [ ] cricket
  * [ ] basketball
  * [ ] baseball
  * [ ] football
  * [ ] golf
  * [ ] boxing
  * [ ] nascar
  * [ ] horse racing

**Comment by pasky** [comment]
Also depends on #8 so that we can determine the JSON format based on that etc.

**Comment by pasky** [comment]
Low priority for now.
